.gitignore

```diff
@@ -33,6 +33,13 @@ libtool
 *~
 *.log
 *.trs
+
+coverage/
```

Comment:
What do you do to get this? I just write gcovr output into a coverage.html. By the way, could you also add *.gcov?

Reply:
Oh yes, this was left over from the earlier PR. And yes, I'll add .gcov.

Reply:
Ok, I ported that one change that suggests to create coverage reports in a `coverage` subdir from #939 over to here. I think that one is still useful for users.